### PR TITLE
Fixed claim rewards visual bugs

### LINKED ISSUE #5377: Staking visual interaction bugs
### Expected behavior

- `Claim rewards` should not have `0` as text while loading

### Steps to reproduce
Screenshot 2023-10-12 at 1 23 47 PM

### Which version(s) does this affect? (Environment, OS, etc...)
3.0.0-rc.2

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #5377, #5383 and #5385

### How was it solved?

- [x] Fixed claim rewards loading text
- [x] Fixed claim rewards trigger
- [x] Fixed multiple claim rewards popup

### How was it tested?

- Visually
- Unit tests

### CODE REVIEW

File: src/modules/common/components/notification/rewardsNotification/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      transactions: { rewards },
+    },
+  } = useContext(ApplicationBootstrapContext);
+  const notification = rewards.length && BigInt(rewards[0]?.reward || 0) > BigInt(0);
+
+  useEffect(() => {
+    if (notification) {
+      toast.info(
+        <div className={styles.rewardInfo}>
+          <p>You have an unclaimed reward of your stakes.</p>
```
Comment (ManuGowda) [inline src/modules/common/components/notification/rewardsNotification/index.js]:
```suggestion
          <p>You have an unclaimed reward for your stakes.</p>
```

**Review by ManuGowda — DISMISSED** [review]

File: src/modules/common/components/notification/rewardsNotification/index.js (hunk 1 above)
Comment (ikem-legend) [reply src/modules/common/components/notification/rewardsNotification/index.js]: Fixed

File: src/modules/common/components/notification/rewardsNotification/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      transactions: { rewards },
+    },
+  } = useContext(ApplicationBootstrapContext);
+  const notification = rewards.length && BigInt(rewards[0]?.reward || 0) > BigInt(0);
+
+  useEffect(() => {
+    if (notification) {
+      toast.info(
+        <div className={styles.rewardInfo}>
+          <p>You have an unclaimed reward for your stakes.</p>
```
Comment (eniolam1000752) [inline src/modules/common/components/notification/rewardsNotification/index.js]: needs translation here

File: src/modules/common/components/notification/rewardsNotification/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          <DialogLink component="claimRewardsView" className={styles.rewardLink}>
+            Claim rewards
```
Comment (eniolam1000752) [inline src/modules/common/components/notification/rewardsNotification/index.js]: and here to

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]

File: src/modules/common/components/notification/rewardsNotification/index.js (hunk 3 above)
Comment (ikem-legend) [reply src/modules/common/components/notification/rewardsNotification/index.js]: Fixed

File: src/modules/common/components/notification/rewardsNotification/index.js (hunk 2 above)
Comment (ikem-legend) [reply src/modules/common/components/notification/rewardsNotification/index.js]: Fixed

**Review by eniolam1000752 — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**